src/lib/services/collection.service.ts

```diff
@@ -180,18 +181,10 @@ export class CollectionService {
   };
 
   public validateImportCollectionURL = async (url = "") => {
-    const response = await makeRequest(
-      "GET",
-      url,
-      {
-        headers: {
-          ...getAuthHeaders(),
-          "Content-type": ContentTypeEnum["application/json"],
-        },
-      },
-      true,
+    return createApiRequest(
+      [url, `GET`, `Accept=*/*&Connection=keep-alive`, ``, `TEXT`],
+      `UNTRACKED-96bff9dc-299e-4c71-a0d8-562dee34d572`,
```

Comment:
what's the meaning of this?
`UNTRACKED-***`

Reply:
Oh not required, removed